spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/servicebus/outbound/ServiceBusMessageHandler.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.integration.servicebus.outbound;
+
+import com.microsoft.azure.servicebus.IMessage;
+import com.microsoft.azure.spring.integration.core.AbstractAzureMessageHandler;
+import com.microsoft.azure.spring.integration.core.SendOperation;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+
+import java.nio.charset.Charset;
+import java.util.UUID;
+
+/**
+ * Outbound channel adapter to publish messages to Azure Service Bus topic and queue.
+ *
+ * @author Warren Zhu
+ */
+public class ServiceBusMessageHandler extends AbstractAzureMessageHandler<IMessage> {
+
+    public ServiceBusMessageHandler(String destination, SendOperation<IMessage> sendOperation) {
+        super(destination, sendOperation);
+    }
+
+    @Override
+    public IMessage toAzureMessage(Message<?> message) {
+        com.microsoft.azure.servicebus.Message serviceBusMessage;
```

Comment:
import com.microsoft.azure.servicebus.Message

Reply:
We already have Spring defined `Message`, so have to use fully qualified name here.